addons/api/tests/unit/serializers/host-test.js

```diff
@@ -4,19 +4,48 @@ import { setupTest } from 'ember-qunit';
 module('Unit | Serializer | host', function (hooks) {
   setupTest(hooks);
 
-  test('it exists', function (assert) {
-    let store = this.owner.lookup('service:store');
-    let serializer = store.serializerFor('host');
-
-    assert.ok(serializer);
+  test('it serializes host with type static as expected', function (assert) {
+    assert.expect(1);
+    const store = this.owner.lookup('service:store');
+    const record = store.createRecord('host', {
+      name: 'Host 1',
+      compositeType: 'static',
+      description: 'Description',
+      host_catalog_id: '123',
+      version: 1,
+      address: '188e:68a9:b342:c05c:2595:2f46:499c:759f',
+    });
+    assert.deepEqual(record.serialize(), {
+      name: 'Host 1',
+      description: 'Description',
+      type: 'static',
+      host_catalog_id: '123',
+      version: 1,
+      attributes: {
+        address: '188e:68a9:b342:c05c:2595:2f46:499c:759f',
+      },
+    });
   });
 
-  test('it serializes records', function (assert) {
-    let store = this.owner.lookup('service:store');
-    let record = store.createRecord('host', {});
-
-    let serializedRecord = record.serialize();
+  test('it serializes host with type plugin as expected', function (assert) {
```

Comment:
We don't need to test serialisation of dynamic hosts because they are read-only.  Only normalisation is important for these.

Reply:
I will add the `normalisation` tests.